src/promptflow/tests/test_configs/eager_flows/chat-basic/flow.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+from promptflow.tracing import trace
+from promptflow.core import AzureOpenAIModelConfiguration, Prompty
+
+BASE_DIR = Path(__file__).absolute().parent
+
+
+@dataclass
+class Result:
+    answer: str
+
+
+class ChatFlow:
+    def __init__(self, model_config: AzureOpenAIModelConfiguration):
```

Comment:
some more test cases (I think we already have some in current test_configs)? Like `AzureOpenAIConnection` is part of init parameters

Reply:
Flex flow build currently does not support connection.